src/Modules/OrchardCore.Commerce/Views/PricePart.cshtml

```diff
@@ -1,6 +1,26 @@
 @model PricePartViewModel
 
-@if (Model.CurrentDisplayCurrency.CurrencyIsoCode == Model.PriceCurrency)
-{
-<div class="product-price">@Model.Price.ToString()</div>
+@{
+    var currentDisplayCurrencyIsoCode = Model.CurrentDisplayCurrency.CurrencyIsoCode;
+    var modelPriceCurrency = Model.PriceCurrency;
+    var modelPrice = Model.Price.ToString();
```

Comment:
do you actually need the ToString? I think that's what both usages should do anyway.